### ISSUE TITLE
Compatibility with Idea-15

### ISSUE THREAD

**Issue by abrighton** [orfjackal/idea-sbt-plugin#111]
It would be great if this plugin worked on Intellij Idea 15.
Currently it seems to be incompatible.

**Comment by timothymschier** [comment]
+1 - as a new user downloading the now current version of IntelliJ this was really frustrating!

**Comment by nicmon** [comment]
+1

**Comment by krivachy** [comment]
+1

**Comment by kirillkh** [comment]
+1

**Comment by skolberg** [comment]
+1
Am 04.11.2015 11:38 vorm. schrieb "kirillkh" <notifications@github.com>:

**Comment by westernsam** [comment]
+1 the build task makes running tests in intellij bearable. I really like this,

**Comment by fnouama** [comment]
+1

**Comment by realzeus** [comment]
+1

**Comment by panCtrlV** [comment]
+1

**Comment by lvitaly** [comment]
+1

**Comment by lanceon** [comment]
+1
That's much better than running sbt in a separate console.

**Comment by JohnColanduoni** [comment]
For what it's worth, I simply compiled the plugin with IDEA 15 and it's running fine for me.

**Comment by kirillkh** [comment]
@JohnColanduoni Would you care to share it?

**Comment by lanceon** [comment]
Hi All, 
Succeeded with building the plugin from master (built within IDEA 15, JDK 1.8).
The only change was to set the path to IDEA libs/sdk to a current installation (IDEA 15 Ultimate).

Zip is here, use "Install plugin from disk":
https://drive.google.com/file/d/0Byve6XhXQAwVVjR5R29sVjNkV0k/view?usp=sharing

You'll see the message "The following plugin is ready to update: SBT", just ignore it.

**Comment by kirillkh** [comment]
Thanks lanceon!

**Comment by abrighton** [comment]
That works, thanks. (Just need to ignore Idea's suggestion to update the SBT plugin, since that will re-install the incompatible version again...)

**Comment by sudheer594** [comment]
Thanks lanceon

**Comment by DTVD** [comment]
Thanks lanceon! You save my day

**Comment by 178inaba** [comment]
Thanks lanceon!!

**Comment by vicsstar** [comment]
Thank you Lanceon! :100:

**Comment by pfn** [comment]
```
git clone https://github.com/orfjackal/idea-sbt-plugin.git
mvn -Didea.version=143.381 -Didea.home="c:\Program Files (x86)\JetBrains\IntelliJ IDEA 15.0" package
```

Substitute the path to IDEA 15.0 for your computer and install the resulting `sbt-dist/target/idea-sbt-plugin-1.7.1-SNAPSHOT.zip` into your IDEA. Done.

**Comment by rgobbelquixey** [comment]
+1

**Comment by ajozwik** [comment]
+1

**Comment by dphans** [comment]
+1

**Comment by Teudimundo** [comment]
+1 
plus: The link given by @lanceon is no longer working

**Comment by lanceon** [comment]
File was moved away accidentally, sorry.
Here it is in case someone needs the build: https://goo.gl/JDfgC1

**Comment by KuchiVanDerK** [comment]
Intellij 15 always wants to update from v. 1.7.0 -> 1.7.0 which on every startup. Which doesn't work.  
IntelliJ 15.0.1 Build 143.382, Ubuntu 14.04 LTS

**Comment by depend** [comment]
+1 and thanks lanceon!

**Comment by andrey-feoktistov** [comment]
+1

**Comment by Sarseth** [comment]
+1. This plugin started to be useless.

**Comment by pfn** [comment]
Please stop spamming +1 and me too

**Comment by retronym** [comment]
Sorry about the slow response on this ticket, and thanks to those who have shared the workaround.

I've just submitted a pull request with the changes that I think will support IDEA 15. I'd appreciate some help in testing this. If you'd like to help, jump over to https://github.com/orfjackal/idea-sbt-plugin/pull/112, install the binary of the plugin that I've linked to there, and comment over on that PR to let me know whether its working as you'd like.
